apps/playground/src/app/shared/theme-selector/theme-selector.component.ts

```diff
@@ -83,7 +83,7 @@ export class SkyThemeSelectorComponent implements OnInit {
         this.themeName.set(previousSettings.themeName);
         this.themeMode.set(previousSettings.themeMode);
         this.themeSpacing.set(previousSettings.themeSpacing);
-        this.modernV2Enabled.set(previousSettings.modernV2Enabled);
+        this.modernV2Enabled.set(!!previousSettings.modernV2Enabled);
```

Comment:
Code examples has a little stricter of Typescript/Angular rules. I made changes there to satisfy those and just ported those into the playground version as well.